sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -2186,6 +2186,24 @@ object functions {
   }
 
   //////////////////////////////////////////////////////////////////////////////////////////////
+  // Collection functions
+  //////////////////////////////////////////////////////////////////////////////////////////////
+
+  /**
+   * Returns length of array or map
+   * @group collection_funcs
```

Comment:
i think you need to update the doc in the beginning of this file to define this new group